waf_error_page.html

```diff
@@ -1,145 +1,165 @@
-    <div class="u-fw-b">System information</div>
-    <div>{{ERROR-CODE}} {{ERROR-ID}}</div>
-  </div>
-
-
-    </main>
-  </div>
-</div>
-
-
-        </div>
-      </div>
-
-      <footer class="page__footer">
-
-          <div class="footer page__footer" role="contentinfo">
-  <div class="container">
-    <div class="pluto">&copy; <a class="footer__link" href="https://www.ons.gov.uk/" target="_blank">Office for National Statistics 2016</a></div>
-  </div>
-</div>
-
-      </footer>
-
-    </div>
-
-
-
-
-
-    <!--[if gte IE 9]><!-->
-
-      <script src="https://eq.ons.gov.uk/s/js/bundle.js"></script>
-
-  	<!--<![endif]-->
-
-    </body>
-</html>
\ No newline at end of file
+<!DOCTYPE html>
+<!--[if lt IE 7]>      <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9 lt-ie8 lt-ie7"> <![endif]-->
+<!--[if IE 7]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9 lt-ie8"> <![endif]-->
+<!--[if IE 8]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10 lt-ie9"> <![endif]-->
+<!--[if IE 9]>         <html lang="en-gb" dir="ltr" class="no-js lt-ie10"> <![endif]-->
+<!--[if gt IE 9]><!-->
+<html lang="en-gb" dir="ltr" class="no-js">
+  <!--<![endif]-->
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>Error (Gateway)</title>
+    <meta content="" name="description" />
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <link rel="shortcut icon" href="https://eq.ons.gov.uk/s/img/favicon.ico" />
+    <!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.min.js"></script>
+    <![endif]-->
+    <!--[if gt IE 8]><!-->
+    <script>
+      document.documentElement.className = document.documentElement.className.replace(
+        "no-js",
+        "has-js"
+      );
+    </script>
+    <!--<![endif]-->
+
+    <!--[if (gt IE 9) | (IEMobile)]><!-->
+    <link
+      href="https://cdn.ons.gov.uk/sdc/v2.0.0/css/responsive.css"
+      rel="stylesheet"
+    />
+    <!--<![endif]-->
+    <!--[if (lte IE 9) & (!IEMobile)]>
+      <link
+        href="https://cdn.ons.gov.uk/sdc/v2.0.0/css/fixed.css"
+        rel="stylesheet"
+      />
+    <![endif]-->
+    <!-- Google Analytics -->
+    <script>
+      (function(i, s, o, g, r, a, m) {
+        i["GoogleAnalyticsObject"] = r;
+        (i[r] =
+          i[r] ||
+          function() {
+            (i[r].q = i[r].q || []).push(arguments);
+          }),
+          (i[r].l = 1 * new Date());
+        (a = s.createElement(o)), (m = s.getElementsByTagName(o)[0]);
+        a.async = 1;
+        a.src = g;
+        m.parentNode.insertBefore(a, m);
+      })(
+        window,
+        document,
+        "script",
+        "https://www.google-analytics.com/analytics.js",
+        "ga"
+      );
+
+      ga("create", "", "auto");
+      ga("set", "anonymizeIp", true);
+      ga("send", "pageview");
+    </script>
+    <!-- End Google Analytics -->
```

Comment:
I think we should remove the Google Analytics code. The WAF error page is for all Survey Data Collection products and there is no combined Google Analytics account we could use here.